<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mDateCheckDlg</class>
 <widget class="QDialog" name="mDateCheckDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Query Item:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mSelDb">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>45</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>SN</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_sn">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QPushButton" name="mCheckBtn">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>Query</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mTotalCount">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="prefix">
           <string>tatol:</string>
          </property>
          <property name="maximum">
           <number>1000000000</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Current Page:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mPage">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>45</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mExportBtn">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>Export</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
